Makefile

```diff
@@ -117,7 +117,7 @@ dev: .example-image dist
 			&& jupyter notebook --ip=* --no-browser"
 
 test: VM_WORKDIR=/src/toree-kernel
-test:
+test: jupyter-tests
```

Comment:
don't combine test and system test... keep them separate. Leave `test` as is and create a `system-test` target.